allure-web/.gitignore

```diff
@@ -3,3 +3,4 @@ coverage
 etc
 node_modules
 node
+yarn.lock
```

Comment:
you should commit this file as mentioned in https://yarnpkg.com/en/docs/yarn-lock#toc-check-into-source-control